Move Apartment and City model to apartments app

tamirho opened the pull request:
- Move the `Apartment` and `City` data models from the `users` app to the `apartments` app
- Add tests for `Apartment` creation and data retrieval from DB
- Alter all migrations and test_data files in the `users` and `contacts` app.
- Add new migrations to the `apartments` model.

Fix #71 
Fix #68 

**Note:**
This PR is dependent on #67  PR

simzacks commented on apartments/apps.py:
```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
```
this file is in PR #67

simzacks commented on users/admin.py:
```diff
@@ -1,5 +1,5 @@
 from django.contrib import admin
-from .models import User, Apartment, City, Hobby, Seeker
+from .models import User, Hobby, Seeker
```
I'm pretty sure that you can register models from multiple apps into a single admin. That way you have one place to go instead of an admin for each app.
This makes sense when the apps are all part of a single system.

tamirho replied:
Ok, this really sounds better than the current state.
In which directory do you recommend putting it?

tamirhouri committed:
Move Apartment and City model

- Move Apartment and City model to the apartments app.
- Create new migration files for that models.

tamirhouri committed:
Alter users & contacts app to run with apartments

- Remove Apartment and City models from users app.
- Alter users/migrations/0006_test_data imports.
- Alter contacts/models.py imports.

tamirhouri committed:
Adding tests to Apartment model

- Add tests for apartment creation and data retrieval from DB

OrAsaf reviewed (approved):
LGTM
